lib/mongoid/embedded_errors/version.rb

```diff
@@ -1,4 +1,6 @@
+# frozen_string_literal: true
+
 module Mongoid; end
 module Mongoid::EmbeddedErrors
-  VERSION = '2.0.1'.freeze
+  VERSION = '3.0.0'
```

Comment:
Given that we're not breaking functionality, let's increase version to 2.1.0

Reply:
Scratch this, we already have 2.1.x on rubygems somewhow. Let's bump this up to 3.0.0 as we're removing support for Mongoid 3.